src/main/java/mate/academy/exception/AuthenticationException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.exception;
+
+public class AuthenticationException extends RuntimeException {
```

Comment:
![image](https://github.com/mate-academy/hibernate-user-service-hw/assets/20326858/1584965c-2529-4514-8b42-a1fc97e36380)

Reply:
Missed this point. Corrected, thanks!